aiobotocore/httpsession.py

```diff
@@ -205,12 +205,10 @@ async def send(self, request):
                 request=request,
                 endpoint_url=request.url
             )
-        except ServerTimeoutError as e:
-            raise ConnectTimeoutError(endpoint_url=request.url, error=e)
```

Comment:
we still do need to support ConnectTimeoutError: https://github.com/boto/botocore/blob/develop/botocore/httpsession.py#L478. so probably need something a little more refined

Reply:
Digging through the source code, it looks like we can differentiate by looking at the message:
- Connection timeout - https://github.com/aio-libs/aiohttp/blob/8e92c4c7c305f914c17f41539da20822ea7baddf/aiohttp/client.py#L505
- Read timeout - https://github.com/aio-libs/aiohttp/blob/8e92c4c7c305f914c17f41539da20822ea7baddf/aiohttp/client_proto.py#L197

How about:

```python
if str(e).lower().startswith('connect'):
    raise ConnectTimeoutError(...)
else:
    raise ReadTimeoutError(...)
```